server/models/idea.js

```diff
@@ -0,0 +1,32 @@
+const mongoUtil = require('../helpers/mongoUtil.js');
+const ObjectID = require('mongodb').ObjectID;
+
+exports.all = function(callback) {
+	mongoUtil.getDb().collection('ideas').find().toArray(function(err, result) {
```

Comment:
Try to be consistent with style. I notice in other files you're using 2 spaces, but here, you're using a hard tab. For frontend code, I'm using eslint for verifying good code style. Maybe consider using eslint on backend code too?